Source/WebCore/html/HTMLLabelElement.cpp

```diff
@@ -151,15 +152,15 @@ void HTMLLabelElement::defaultEventHandler(Event& event)
             return;
         }
 
-        processingClick = true;
+        SetForScope processingClick(m_processingClick, true);
 
         control->dispatchSimulatedClick(&event);
 
         document().updateLayoutIgnorePendingStylesheets();
         if (control->isMouseFocusable())
             control->focus({ { }, { }, { }, FocusTrigger::Click, { } });
 
-        processingClick = false;
+        SetForScope processingClick(m_processingClick, false);
```

Comment:
Oh you don't need this second SetForScope. A neat thing about SetForScope is that it would restore the original value in its destructor.

Reply:
Also should I add value of { false } in header (HTMLLabelElement.h) and then remove global variable in HTMLLabelElement.cpp?

Reply:
Sure, initializing in the header file makes sense to me.

Reply:
Updated commit message and also did all changes. Now let's see.